packages/cli/src/commands/remove-plugin/file-mocks.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export const pluginsFileContent = `    
+export { plugin as WelcomePlugin } from '@backstage/plugin-welcome';
+export { plugin as LighthousePlugin } from '@backstage/plugin-lighthouse';`;
+
+export const codeownersFileContent = `
+*                                           @spotify/backstage-core
+/docs/features/techdocs                     @spotify/techdocs-core
+/plugins/cost-insights                      @spotify/silver-lining
+`;
+
+export const packageFileContent = {
+  name: 'example-app',
+  version: '0.1.1',
+  dependencies: {},
+  devDependencies: {},
+  scripts: {},
+};
```

Comment:
Maybe I can prefix the mocked files content by a keyword to be more explicit when we use it inside the test file ? 
`packageFileContent `  -> `mockedPackageFileContent `